Automattic/jetpack
# JS Components: Convert AdminPage component to TypeScript

#### Changes proposed in this Pull Request:
* This PR converts `AdminPage` component to TypeScript.
* It also converts the `AdminPage` stories to TS

#### Other information:


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Checkout the PR
* Run `cd projects/js-packages/storybook && pnpm run storybook:dev`
* Confirm that all the stories of `AdminPage` component work as expected
* Check the consumers of `AdminPage` and related components and confirm that they work as expected
* Check for any regression

<img width="1425" alt="Screenshot 2022-08-02 at 10 51 42 AM" src="https://user-images.githubusercontent.com/18226415/182297798-cfe60766-bc20-4196-96ad-f1f734e29958.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202276314263994
